File: spring-boot-postgresql/src/main/java/com/akvone/data_jpa/entities/CityEntity.java
package com.akvone.data_jpa.entities;

import java.util.List;
import javax.persistence.CascadeType;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
public class CityEntity {

  @Id
  @GeneratedValue
  Long id;

  String name;

  @OneToMany(cascade = {CascadeType.ALL})
  List<StreetEntity> streets;
}

File: sandboxes/spring-boot-postgresql/src/main/kotlin/com/akvone/data_jpa/entities/CityEntity.kt
package com.akvone.data_jpa.entities

import javax.persistence.*

@Entity
class CityEntity {
    @Id
    @GeneratedValue
    var id: Long? = null
    var name: String? = null

    @OneToMany(cascade = [CascadeType.ALL])
    var streets: List<StreetEntity> = emptyList()
}